<!-- http://wiki.ros.org/roslaunch/XML/node -->
<launch>
   <node name="optitrack_motive_2_client_node" pkg="optitrack_motive_2_client" type="optitrack_motive_2_client_node" output="screen">
      <!--<rosparam file="$(find mypackage)/config/example.yaml" command="load" />-->
      <param name="MyIPAddress" type="string" value="192.168.50.170" />
      <param name="ServerIPAddress" type="string" value="192.168.50.79" />
      <!--备注-->
   </node>

   <node name="SAPcontrol" pkg="learning_com" type="SAPcontrol" output="screen">
      <!--<rosparam file="$(find mypackage)/config/example.yaml" command="load" />-->
      <param name="flag" type="int" value="1" />
      <!--flag =1 为圆周运动，2为余弦运动 -->     
      <param name="Slip_com" type="int" value="1" />
      <!--如果为0就是不滑转补偿，1是滑转补偿，后续考虑把参数也放进来-->
      <param name="SMC_on" type="int" value="2" />
      <!--如果为0就是开环跑，1是普通SMC，2是自适应SMC-->

   </node>
</launch>
